### PR TITLE
Add a paragraph about EE permission application on events

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**
New paragraph to explain how we manage EE permission application to send or not a product event.

### CODE REVIEW

File: content/events-api/more-about-events.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
\ No newline at end of file
+
+## EE permission application
+
+::: warning
+The following paragraph is for EE customers.
+:::
+
+You may ask yourself how EE permissions apply to the event subscription. 
+Indeed, Akeneo PIM uses the permissions related to the connection user group to filter what it has to send or not.
```
Comment (camillebourjade) [inline content/events-api/more-about-events.md:10]:
```suggestion
Indeed, Akeneo PIM uses the permissions related to the connection user group to filter what it is supposed to send or not. 
```

File: content/events-api/more-about-events.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
 # More about events
-If you need details about event format, please take a look to our [Events API Reference](/events-reference/events-reference-serenity/products.html)
\ No newline at end of file
+
+## EE permission application
```
Comment (camillebourjade) [inline content/events-api/more-about-events.md:3]:
```suggestion
## Enterprise Edition (EE) permission application
```

File: content/events-api/more-about-events.md [hunk 3]
```diff
@@ -1,2 +1,26 @@
...
+:::
+
+You may ask yourself how EE permissions apply to the event subscription. 
+Indeed, Akeneo PIM uses the permissions related to the connection user group to filter what it has to send or not. 
+
+That's why when you enable the event subscription & setup your connection permissions, you have to keep in mind the following information:
+- **Akeneo PIM sends events on products your connection can view.** 
+
+To receive an event, your connection user group must, at least, view one of the product categories.  
+N.b. If the product has no category at all, the connection will receive the event.
```
Comment (camillebourjade) [inline content/events-api/more-about-events.md:16]:
```suggestion
N.b. By default, if the product is `Unclassified`, the connection will receive the event. 
```

File: content/events-api/more-about-events.md [hunk 4]
```diff
@@ -1,2 +1,26 @@
...
+
+That's why when you enable the event subscription & setup your connection permissions, you have to keep in mind the following information:
+- **Akeneo PIM sends events on products your connection can view.** 
+
+To receive an event, your connection user group must, at least, view one of the product categories.  
+N.b. If the product has no category at all, the connection will receive the event. 
+
+- **Akeneo PIM sends product events even if the updated values aren't accessible by the connection.** 
+
+For example, Julia updates the ERP code of a product. Your e-commerce connection doesn't have access to the attribute group containing the ERP code but will receive an event anyway.
```
Comment (camillebourjade) [inline content/events-api/more-about-events.md:20]:
```suggestion
Here are a couple of examples to highlight different scenarios
### Permissions based on attribute groups
The ERP code of a product is updated. Your e-commerce connection doesn't have access to the attribute group containing the ERP code but will receive an event anyway. In the event payload, there will be no mention of the newly updated ERP code, to respect the permission granted at the attribute group level. 
```

File: content/events-api/more-about-events.md [hunk 5]
```diff
@@ -1,2 +1,26 @@
...
+That's why when you enable the event subscription & setup your connection permissions, you have to keep in mind the following information:
+- **Akeneo PIM sends events on products your connection can view.** 
+
+To receive an event, your connection user group must, at least, view one of the product categories.  
+N.b. If the product has no category at all, the connection will receive the event. 
+
+- **Akeneo PIM sends product events even if the updated values aren't accessible by the connection.** 
+
+For example, Julia updates the ERP code of a product. Your e-commerce connection doesn't have access to the attribute group containing the ERP code but will receive an event anyway.  
+Another example, Julia updates the Italian description of the product. Your e-commerce connection doesn't have access to the Italian locale but will receive an event anyway.
```
Comment (camillebourjade) [inline content/events-api/more-about-events.md:21]:
```suggestion
### Permissions based on locales
Another example, the translation team updates the Italian description of the product. Your e-commerce connection doesn't have access to the Italian locale but will receive an event anyway. In the event payload, there will be no mention of the newly updated Italian description, to respect the permission granted at the locale level. 
```

File: content/events-api/more-about-events.md [hunk 6]
```diff
@@ -1,2 +1,26 @@
...
+- **Akeneo PIM sends events on products your connection can view.** 
+
+To receive an event, your connection user group must, at least, view one of the product categories.  
+N.b. If the product has no category at all, the connection will receive the event. 
+
+- **Akeneo PIM sends product events even if the updated values aren't accessible by the connection.** 
+
+For example, Julia updates the ERP code of a product. Your e-commerce connection doesn't have access to the attribute group containing the ERP code but will receive an event anyway.  
+Another example, Julia updates the Italian description of the product. Your e-commerce connection doesn't have access to the Italian locale but will receive an event anyway.  
+However, Akeneo PIM applies the EE permissions to the data it sends. So, the payload doesn't contain values the connection can't access.
```
Comment (camillebourjade) [inline content/events-api/more-about-events.md:22]:
```suggestion
```

**Review by camillebourjade — COMMENTED** [review]
English Fix

**Review by tseho — APPROVED** [review]

**Pull request merged**